src/sources/file.rs

```diff
@@ -108,32 +109,39 @@ pub struct FileConfig {
 
     #[configurable(derived)]
     #[serde(default)]
```

Comment:
Do we want to create a helper function to return an explicit default ReadFromConfig? I think that's a part of the new pattern we're adopting according to #14998

Reply:
Addressed in [0e7c38b](https://github.com/vectordotdev/vector/pull/16099/commits/0e7c38bf4f326cfe4575ba418a6da0996b4a8658)